<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>905</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>950</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>X CODER</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(4, 15, 15)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="Titles">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="Title">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>80</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 210, 105);</string>
        </property>
        <property name="text">
         <string>X CODER</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Link">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;tg://resolve?domain=XcoderBS&quot;&gt;&lt;span style=&quot; font-size:14pt; text-decoration: underline; color:#007af4;&quot;&gt;https://t.me/XcoderBS&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="Decoding">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255,255, 60);
border: 2px solid rgba(252, 255, 255, 80);
border-radius: 10px;</string>
      </property>
      <layout class="QVBoxLayout" name="Decode">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="Decode_txt">
         <property name="font">
          <font>
           <pointsize>80</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none;
color: white;
border: 0px;</string>
         </property>
         <property name="text">
          <string>DECODE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="btns">
         <item>
          <widget class="QPushButton" name="DecodeFile">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>65</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>40</pointsize>
             <bold>true</bold>
             <hintingpreference>PreferDefaultHinting</hintingpreference>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {color: white;}
QPushButton:hover:!pressed
{
  background-color: rgba(255, 255,255, 80);
  border: 2px solid rgba(252, 255, 255, 210);
}
QPushButton:pressed
{
  background-color: rgba(255, 255,255, 100);
  border: 2px solid rgba(252, 255, 255, 220);
}</string>
           </property>
           <property name="text">
            <string>FILE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DecodeFolder">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>65</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>40</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {color: white;}
QPushButton:hover:!pressed
{
  background-color: rgba(255, 255,255, 80);
  border: 2px solid rgba(252, 255, 255, 210);
}
QPushButton:pressed
{
  background-color: rgba(255, 255,255, 100);
  border: 2px solid rgba(252, 255, 255, 220);
}</string>
           </property>
           <property name="text">
            <string>FILES</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="Encoding">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255,255, 60);
border: 2px solid rgba(252, 255, 255, 80);
border-radius: 10px;</string>
      </property>
      <layout class="QVBoxLayout" name="Encode">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="EncodeText">
         <property name="font">
          <font>
           <pointsize>80</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none;
color: white;
border: 0px;</string>
         </property>
         <property name="text">
          <string>ENCODE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="btns_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="EncodeFile">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>65</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>40</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {color: white;}
QPushButton:hover:!pressed
{
  background-color: rgba(255, 255,255, 80);
  border: 2px solid rgba(252, 255, 255, 210);
}
QPushButton:pressed
{
  background-color: rgba(255, 255,255, 100);
  border: 2px solid rgba(252, 255, 255, 220);
}</string>
           </property>
           <property name="text">
            <string>FOLDER</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="EncodeFolder">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>65</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>40</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {color: white;}
QPushButton:hover:!pressed
{
  background-color: rgba(255, 255,255, 80);
  border: 2px solid rgba(252, 255, 255, 210);
}
QPushButton:pressed
{
  background-color: rgba(255, 255,255, 100);
  border: 2px solid rgba(252, 255, 255, 220);
}</string>
           </property>
           <property name="text">
            <string>FOLDERS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255,255, 60);
border: 2px solid rgba(252, 255, 255, 80);
border-radius: 10px;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>65</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none;
color: white;
border: 0px;</string>
         </property>
         <property name="text">
          <string>LANGUAGE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
    color:white;
    margin: 5px;
}

QComboBox::drop-down {
    width: 40px;
}
QComboBox:editable {
    background: white;
}</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Русский</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="About">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover:!pressed
{
  background-color: rgba(255, 255,255, 100);
  border: 2px solid rgba(252, 255, 255, 210);
}
QPushButton:pressed
{
  background-color: rgba(255, 255,255, 110);
  border: 2px solid rgba(252, 255, 255, 220);
}
QPushButton
{
  color: white;
  background-color: rgba(255, 255, 255,90);
  border: 2px solid rgba(252, 255, 255, 100);
  border-radius: 10px;
}
</string>
      </property>
      <property name="text">
       <string>ABOUT</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
